ui/src/RecentFilesDrawer/WorkflowListItem.tsx

```diff
@@ -60,6 +67,7 @@ export default function WorkflowListItem({
         {/* </Stack> */}
       </Box>
 
+      <IconCopy cursor={"pointer"} onClick={handleCopyItem} />
```

Comment:
can you make a right click menu that when user right clicks a file item, the menu pop up, and it will contain the "duplicate" option